src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ CHANGELOG
 6.4
 ---
 
+ * Add `Mac` constraint
```

Comment:
MacAddress?

Reply:
Changed back to `Mac`, `Ip` is also not called `IpAddress`, to be a bit consistent. Also there are no different meanings of that word (I think). But we can change that to `MacAddress` or kick it completely out, I'm fine with both that :-)